src/System Application/App/Rest Client/src/ExceptionHandling/RestClientExceptionBuilder.Codeunit.al

```diff
@@ -0,0 +1,28 @@
+namespace System.RestClient;
+
+codeunit 2362 "Rest Client Exception Builder"
+{
```

Comment:
Public Codeunits should be explict with the Access property.
And public Codeunits require a documentation header.

Reply:
Didn't know about the requirement of Access = Public on all public codeunits. It's a default property anyway.
Other objects didn't have this as well. So I've now added it to all objects for the sake of unity and clarity.

Reply:
At least this pattern was used when the emphasized the Facade Pattern in the System application. 

Will you add an XML Summary to the enum values and the procedures in the new Exception Codeunit, too?
(I think the term "documentation header" was a little bit unclear)

Reply:
Added documentation to the functions and enum.

Reply:
I don't want to annoy you, since I really appreciate your contribution, but I think that the XML Summary should contain the lines for parameters.

Reply:
Eh... yes of course... Done!